### PR TITLE
Reset button

### PR DESCRIPTION
**Pull request by TigranBoomTech**

### CODE REVIEW

File: src/Reset/Reset.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div className={'boomForm-resetButton__content'}>
+      <button
+        onClick={e => {
+          handleClick(e)
+        }}
+      >
+        {text.length > 10 ? 
+          text.substring(0,10) + "..."  
+        : text}
+      </button>
```
Comment (Tigran0199) [inline src/Reset/Reset.js:19]: lets set button content as a dangerously innerhtml

File: src/Footer/SubmitButton/SubmitButton.js [hunk 2]
```diff
@@ -22,7 +23,8 @@ const SubmitButton = ({
   const { state, actions } = useContext(Context)
   const { values } = state
   const { text, prefix, suffix } = button || { text: 'Submit' }
-  const { name, description, isPrint, onSubmit, onSubmitFailed } = global
+  const { name, description, isPrint, onSubmit, onSubmitFailed, resetButton } = global
+  const { resetText , resetEnabled} = resetButton
```
Comment (Tigran0199) [inline src/Footer/SubmitButton/SubmitButton.js:27]: also lets add callback into resetButton data
if callback function provided call the function and pass as a argument reset function

**Review by Tigran0199 — CHANGES_REQUESTED** [review]

File: src/Reset/Reset.js (hunk 1 above)
Comment (Tigran-Oqroyan) [reply src/Reset/Reset.js]: done

File: src/Footer/SubmitButton/SubmitButton.js (hunk 2 above)
Comment (Tigran-Oqroyan) [reply src/Footer/SubmitButton/SubmitButton.js]: done

**Pull request merged**